Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:19.10
```

Comment:
Any reason why 19.10 instead of 20.04 which is LTS?

Reply:
No reason, just scaffolded the Dockerfile from another project. I've bumped the tag to Ubuntu 20.04.